troubleshooting-2.md

```diff
@@ -0,0 +1,466 @@
+Team Troubleshooting Deliverable 2
+================
+
+There are **11 code chunks with errors** in this Rmd. Your objective is
+to fix all of the errors in this worksheet. For the purpose of grading,
+each erroneous code chunk is equally weighted.
+
+Note that errors are not all syntactic (i.e., broken code)! Some are
+logical errors as well (i.e. code that does not do what it was intended
+to do).
+
+## Exercise 1: Exploring with `select()` and `filter()`
+
+[MovieLens](https://dl.acm.org/doi/10.1145/2827872) are a series of
+datasets widely used in education, that describe movie ratings from the
+MovieLens [website](https://movielens.org/). There are several MovieLens
+datasets, collected by the [GroupLens Research
+Project](https://grouplens.org/datasets/movielens/) at the University of
+Minnesota. Here, we load the MovieLens 100K dataset from Rafael Irizarry
+and Amy Gill’s R package,
+[dslabs](https://cran.r-project.org/web/packages/dslabs/dslabs.pdf),
+which contains datasets useful for data analysis practice, homework, and
+projects in data science courses and workshops. We’ll also load other
+required packages.
+
+``` r
+### ERROR HERE ###
+library(dslabs)
+library(tidyverse)
+```
+
+    ## ── Attaching core tidyverse packages ──────────────────────── tidyverse 2.0.0 ──
+    ## ✔ dplyr     1.1.3     ✔ readr     2.1.4
+    ## ✔ forcats   1.0.0     ✔ stringr   1.5.0
+    ## ✔ ggplot2   3.4.3     ✔ tibble    3.2.1
+    ## ✔ lubridate 1.9.2     ✔ tidyr     1.3.0
+    ## ✔ purrr     1.0.2     
+    ## ── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
+    ## ✖ dplyr::filter() masks stats::filter()
+    ## ✖ dplyr::lag()    masks stats::lag()
+    ## ℹ Use the conflicted package (<http://conflicted.r-lib.org/>) to force all conflicts to become errors
```

Comment:
Some references about [mask stats.](https://stackoverflow.com/questions/39137110/what-does-the-following-object-is-masked-from-packagexxx-mean)